File: In_C_Programming/q4.c
#include <stdio.h>

void get_element(int arr[], int ln)
{

    printf("Enter The Elements: ");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr[i]);
    }
}

int get_Avg(int arr[], int ln)
{

    int sum = 0;
    int avg = 0;

    for (int i = 0; i < ln; i++)
    {
        sum += arr[i];
    }

    avg = sum / ln;

    return avg;
}

int get_Sum(int arr[], int ln)
{

    int sum = 0;

    for (int i = 0; i < ln; i++)
    {
        sum += arr[i];
    }

    return sum;
}

int main()
{

    int size;

    printf("Enter The Array Size: ");
    scanf("%d", &size);

    int arr[size];

    get_element(arr, size);

    int result = get_Sum(arr, size);
    int result2 = get_Avg(arr, size);

    printf("The Sum of the Elements is: %d \n", result);
    printf("The Average of the Elements is: %d \n", result2);

    return 0;
}

File: In_Java_Programming/q4.java
import java.util.Scanner;

public class q4 {

    public static int getAvg(int arr[], int size) {

        int avg = 0, sum = 0;

        for (int i = 0; i < size; i++) {
            sum += arr[i];
        }

        avg = sum / size;

        return avg;
    }

    public static int getSum(int arr[], int size) {

        int sum = 0;

        for (int i = 0; i < size; i++) {
            sum += arr[i];
        }
        return sum;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter The Array Length: ");
        int ln = sc.nextInt();

        int x[] = new int[ln];

        int result = getSum(x, ln);
        int result2 = getAvg(x, ln);

        System.out.println("The Sumation is: " + result);
        System.out.println("The Average is: " + result2);

        sc.close();
    }
}
